src/containers/Accounts/AccountTransactionTable/AccountTransactionTable.tsx

```diff
@@ -11,11 +11,13 @@ import { concatTx } from '../../shared/utils'
 
 import { loadAccountTransactions } from './actions'
 
-export const AccountTxTable = (props) => {
+// There will be fewer props in the next PR when react-query is used instead of redux.
+// It's `any` for now
```

Comment:
Should this have a `TODO:` to remind us to remove the comment once we clean this up?

Reply:
I am removing this in a PR I just drafted.  #410